app/src/main/res/values/strings.xml

```diff
@@ -218,6 +218,7 @@
     <string name="person_profile_report_person">Report Person</string>
     <string name="pictrs_image_upload_image">Upload Image</string>
     <string name="postListing_deleted">Deleted</string>
+    <string name="postListing_removed">Removed</string>
```

Comment:
I *really* want to do away with all these "contextual" translations, but it'd potentially be a lot of new work for translators. `Removed` is the same everywhere.

Reply:
And why can the existing `removed` not be used? 

You can remove existing contextual removed and rename to be generic

Reply:
Thx, will do.

Reply:
just keep the one for post/comment removed deleted separate as that one can be different across languages